source/features/remove-zero-packages.tsx

```diff
@@ -0,0 +1,24 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+function init(): void {
+	const packageElem: HTMLElement | null = select(`.numbers-summary a[href='/${getRepoURL()}/packages']`);
+	if (packageElem && packageElem.innerText.trim().startsWith('0')) {
+		const parentElem = packageElem.parentElement;
+		if (parentElem) {
+			parentElem.remove();
+		}
+	}
+}
+
+features.add({
+	id: __featureName__,
+	description: 'Removing 0 package',
```

Comment:
Can we change this to something like: "Hides the packages tab, when no packages exist."?

Reply:
kk